Lombiq.LoginAsAnybody/Manifest.cs

```diff
@@ -0,0 +1,21 @@
+using OrchardCore.Modules.Manifest;
+using static Lombiq.LoginAsAnybody.Constants.FeatureIds;
+
+[assembly: Module(
+    Name = "Lombiq Login as Anybody",
+    Author = "Lombiq Technologies",
+    Version = "0.0.1",
+    Description = "Module that allows administrators to log in as any user.",
```

Comment:
I think the descriptions are not accurate, since you have to have a site owner permission not an administrator. Same in the [Lombiq.LoginAsAnybody.csproj](https://github.com/Lombiq/Orchard-Login-as-Anybody/pull/3/files#diff-088c92c05f24652165e5ee610ccff5d701e31117d18108e456ea3f2d3b3b1dee).